### ISSUE TITLE
Not downloading course files

### ISSUE THREAD

**Issue by tanushshukla** [Jayapraveen/INE-courses-downloader#28]
Hello,

So, I tried to download one of the course "Azure Pentesting" and it worked just I would expect. Everything downloaded perfectly.

I saw that there are course files attached to it which it did not download.

![image](https://user-images.githubusercontent.com/9604538/127335962-6cd5b801-e2df-4d20-b8b6-7ddbcc85fbd4.png)

Is it expected behaviour?

Thank you for the tool.

**Comment by tanushshukla** [comment]
I have fixed it. They have changed how you get the content files.

I can raise the Pr if you like?

**Comment by dexter101010** [comment]
> I have fixed it. They have changed how you get the content files.
> 
> @Jayapraveen
> I can raise the Pr if you like?

can you improve the script ? Thanks!

**Comment by tanushshukla** [comment]
> > I have fixed it. They have changed how you get the content files.
> > @Jayapraveen
> > I can raise the Pr if you like?
> 
> can you improve the script ? Thanks!

yes, I have fixed two bugs. This and the other one related to slides not downloading in windows.

I can create my own fork and push the changes if the dev is inactive.

**Comment by dexter101010** [comment]
> > > I have fixed it. They have changed how you get the content files.
> > > @Jayapraveen
> > > I can raise the Pr if you like?
> > 
> > 
> > can you improve the script ? Thanks!
> 
> yes, I have fixed two bugs. This and the other one related to slides not downloading in windows.
> 
> I can create my own fork and push the changes if the dev is inactive.

Can you share it somehow with us ?You are on reddit , telegram or skype ?Regards!

**Comment by tanushshukla** [comment]
> > > > I have fixed it. They have changed how you get the content files.
> > > > @Jayapraveen
> > > > I can raise the Pr if you like?
> > > 
> > > 
> > > can you improve the script ? Thanks!
> > 
> > 
> > yes, I have fixed two bugs. This and the other one related to slides not downloading in windows.
> > I can create my own fork and push the changes if the dev is inactive.
> 
> Can you share it somehow with us ?You are on reddit , telegram, discord or skype ?Regards!

yes im on all of these. leave any of your accounts, ill dm you.

**Comment by dexter101010** [comment]
> > > > > I have fixed it. They have changed how you get the content files.
> > > > > @Jayapraveen
> > > > > I can raise the Pr if you like?
> > > > 
> > > > 
> > > > can you improve the script ? Thanks!
> > > 
> > > 
> > > yes, I have fixed two bugs. This and the other one related to slides not downloading in windows.
> > > I can create my own fork and push the changes if the dev is inactive.
> > 
> > 
> > Can you share it somehow with us ?You are on reddit , telegram, discord or skype ?Regards!
> 
> yes im on all of these. leave any of your accounts, ill dm you.

Discord : dexter4242 
tag #3142

Telegram : casperghost39

Reddit : dexter3535

Regards!

**Comment by telus-beep** [comment]
Hi @tanushshukla can you please look at  issue # 26 (You do not have the subscription/pass to access to this course #26)

No one has replied to that.
